napari/components/camera.py

```diff
@@ -54,6 +57,45 @@ def view_direction(self) -> np.ndarray:
         )
         return view_direction
 
+    @view_direction.setter
+    def view_direction(self, view_direction: Tuple[float, float, float]):
+        if (view_direction[0], view_direction[2]) == (0, 0):
+            up_direction = (-1, 0, 0)
+        else:
+            up_direction = (0, -1, 0)
+        self.set_view_direction(
+            view_direction=view_direction, up_direction=up_direction
+        )
+
+    def set_view_direction(
```

Comment:
I think this should be private, otherwise it's a bit redundant with the `view_direction` setter. Can it also get a more complete doc string

Reply:
this has become the only API and others were remove, keeping it public because of that